projects/ziti-console-lib/src/lib/services/validation.service.ts

```diff
@@ -86,6 +86,6 @@ export class ValidationService {
     }
 
     isValidInterceptHost(address) {
-        return /^[^.]+\.[^.]+$/.test(address);
+        return /^(?!\.)(?!.*\.$)([^.]*\.[^.]+)+$/.test(address);
```

Comment:
this should really just be as simple as "contains a period". there are so many rules to get right here, i don't think we should do anything more

Reply:
In that case would it be ok for the following hostnames to be considered valid?
* `.hostname`
* `hostname.`

Reply:
Updated validation logic.